src/tasks/minions/HunterActivity/aerialFishingActivity.ts

```diff
@@ -100,41 +100,32 @@ export const aerialFishingTask: MinionTask = {
 			}
 		}
 
-		await user.addXP({ skillName: SkillsEnum.Fishing, amount: fishXpReceived });
-		await user.addXP({ skillName: SkillsEnum.Hunter, amount: huntXpReceived });
+		const fishXP = await user.addXP({
+			skillName: SkillsEnum.Fishing,
+			amount: fishXpReceived,
+			duration: data.duration
+		});
+		const huntXP = await user.addXP({
+			skillName: SkillsEnum.Hunter,
+			amount: huntXpReceived,
+			duration: data.duration
```

Comment:
see you could add `, minimal: false` to this calls to get the XP/hr.

Reply:
> see you could add `, minimal: false` to this calls to get the XP/hr.

if it's undefined it also gives xp/hr.